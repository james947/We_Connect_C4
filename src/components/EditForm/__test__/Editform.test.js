import { shallow } from 'enzyme'
import React from 'react';
import  EditForm from '../';

describe("<EditForm/>", () => {
    it('renders correctly', () => {
    const data = {
            businessname: '',
            description: '',
            location: '',
            category: ''
    }
      const location = {
          query: {
              business: {
                  state: data
              }
          }
      }   

        const wrapper = shallow(<EditForm onSubmit={()=>{}} onChange={()=>{}} location={location} state={{data:data}} />);
        expect(wrapper).toMatchSnapshot();
    })
})