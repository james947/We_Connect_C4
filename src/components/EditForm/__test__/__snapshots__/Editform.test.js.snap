// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditForm/> renders correctly 1`] = `
<div>
  <Connect(NavBar) />
  <div
    className="auth-form col-md-4"
  >
    <div
      className="panel panel-default"
    >
      <div
        className="panel-body"
      >
        <form
          onSubmit={[Function]}
        >
          <div
            className="form-group"
          >
            <label
              htmlFor="businessName"
            >
              Business Name
            </label>
            <input
              className="form-control"
              name="businessname"
              onChange={[Function]}
              type="text"
              value=""
            />
          </div>
          <div
            className="form-group"
          >
            <label
              htmlFor="businessName"
            >
              Description
            </label>
            <textarea
              className="form-control"
              id="comment"
              name="description"
              onChange={[Function]}
              rows="3"
              value=""
            />
          </div>
          <div
            className="form-group"
          >
            <label
              htmlFor="businessName"
            >
              Category
            </label>
            <div
              className="input-group mb-3"
            >
              <div
                className="input-group-prepend"
              >
                <label
                  className="input-group-text"
                  htmlFor="inputGroupSelect01"
                >
                  Options
                </label>
              </div>
              <select
                className="custom-select"
                name="category"
                onChange={[Function]}
                value=""
              >
                <option
                  selected={true}
                >
                  Choose...
                </option>
                <option
                  value="Electronics"
                >
                  Electronics
                </option>
                <option
                  value="Housing"
                >
                  Housing
                </option>
                <option
                  value="Interior design"
                >
                  Interior design
                </option>
                <option
                  value="Photography"
                >
                  Photography
                </option>
              </select>
            </div>
          </div>
          <div
            className="form-group"
          >
            <label
              htmlFor="email"
            >
              Location
            </label>
            <input
              className="form-control"
              name="location"
              onChange={[Function]}
              type="textArea"
              value=""
            />
          </div>
          <button
            className="btn btn-success btn-block"
            type="submit"
          >
            Submit
          </button>
        </form>
      </div>
    </div>
  </div>
  <Footer />
</div>
`;
